### PR TITLE
Add `matatika` utility

### PR DESCRIPTION
**Pull request by ReubenFrankel**
## Checklist

- [x] Add/update the file in the appropriate folder (`/taps` or `/targets`). The name of the file should match the name of the tap. If there is already one, add a descriptor to the name such as `-search`.
- [x] Add/update the PNG logo image in `/assets/logos/<taps or targets>`. The image name must match the YAML file name.
- [x] Tag `@tayloramurphy` or `@pnadolny13` to flag it for review. Or post to the [#hub](https://meltano.slack.com/archives/C01UGBSJNG5) channel on Meltano slack.

## Reviewer Checklist

- [ ] Validate file against JSON Schema. https://www.jsonschema.net/home is an option.
- [ ] Build website locally and validate everything works.

### CODE REVIEW

**Comment by aaronsteers** [comment]
👀 : https://deploy-preview-1186--meltano-hub.netlify.app/utilities/matatika

File: _data/meltano/utilities/matatika/matatika.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  ```
+  meltano invoke matatika lab
+  ```
+
+  If this is first time this command has been run, container images will be pulled from Docker Hub - be aware that this may take a couple of minutes, depending on the speed of your internet connection.
+  Once the images exist on the host machine, start-up should take no longer than 30s.
+
+  By default, a tab in your default browser will be opened to the Lab automatically at https://localhost:3443.
+pip_url: git+https://github.com/Matatika/matatika-ext@v0.1.0.git
```
Comment (aaronsteers) [inline _data/meltano/utilities/matatika/matatika.yml:47]: repo: https://github.com/Matatika/matatika-ext

File: _data/meltano/utilities/matatika/matatika.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+usage: For help, try `meltano invoke matatika --help` (or `--help` on any subcommand).
```
Comment (aaronsteers) [inline _data/meltano/utilities/matatika/matatika.yml:48]:
```suggestion
pip_url: git+https://github.com/Matatika/matatika-ext@v0.1.0.git
repo: https://github.com/Matatika/matatika-ext
usage: For help, try `meltano invoke matatika --help` (or `--help` on any subcommand).
```

The Repo section in the right pane doesn't seem to be rendering correctly. Trying this to see if it resolves it.

**Comment by ReubenFrankel** [comment]
Ah yes, I wasn't sure `repo` was applicable here since (following convention) it would point to a private Matatika repo. Thanks for correcting @aaronsteers

**Comment by aaronsteers** [comment]
> Ah yes, I wasn't sure `repo` was applicable here since following the convention of other extension utilities, it would have had to point at a private Matatika repo. Thanks for correcting @aaronsteers

Thanks, @ReubenFrankel. Would it make sense to use https://github.com/Matatika/matatika-ce here, or is that different? Either way, I think it is fine to link to the same repo in both places if the main "product" repo is private as you mention.

**Comment by ReubenFrankel** [comment]
> > Ah yes, I wasn't sure `repo` was applicable here since following the convention of other extension utilities, it would have had to point at a private Matatika repo. Thanks for correcting @aaronsteers
> 
> Thanks, @ReubenFrankel. Would it make sense to use https://github.com/Matatika/matatika-ce here, or is that different? Either way, I think it is fine to link to the same EDK repo in both places if the primary repo is private as you mention.

I'll pretend I didn't forget we had this exact discussion internally earlier today... :sweat_smile: Updated it just now, thanks @aaronsteers

**Review by pnadolny13 — APPROVED** [review]
@ReubenFrankel this looks good to me!

**Comment by ReubenFrankel** [comment]
Thanks for the quick merge guys. :+1:

**Pull request merged**